pkg/k8s/watchers/node.go

```diff
@@ -55,11 +51,11 @@ func (k *K8sWatcher) NodesInit(k8sClient *k8s.K8sClient) {
 					var valid bool
 					if node := k8s.ObjToV1Node(obj); node != nil {
 						valid = true
-						if hasAgentNotReadyTaint(node) || !k8s.HasCiliumIsUpCondition(node) {
+						if hasAgentNotReadyTaint(node) {
```

Comment:
`!k8s.HasCiliumIsUpCondition(node)` probably shouldn't be removed, right?

Reply:
hmm, not in the set of my commits, should we add this back?

Reply:
Yes I think so